addons/binding/org.openhab.binding.dlinksmarthome/OSGI-INF/DLinkSmartHomeHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2017 by the respective copyright holders.

```

Comment:
I had ran that previously, that one file must have slipped through when I refactured the binding. Fixed.